docs/storage/storage.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
-title: "Storage"
-description: "Notebook Storage option for Zeppelin"
+title: "Notebook Storage Options for Apache Zeppelin"
+description: Apache Zeppelin has a pluggable notebook storage mechanism controlled by zeppelin.notebook.storage configuration option with multiple implementations. Currently 4 options(Git, S3, Azure and ZeppelinHub) are available in Apache Zeppelin."
```

Comment:
if we keep the number of implementations here - it needs to be updated every time a new one is contributed.
And we have 2 PRs implementing new P2P notebooks storages.